backend/app/Http/Controllers/Api/NewsApiController.php

```
@@ -0,0 +1,20 @@
+use App\Http\Controllers\Controller;
+use App\Models\NewsArticle;
+
+class NewsApiController extends Controller
+{
+
+    public function getNews()
+    {
+        return NewsArticle::paginate();
+    }
+
+    public function getArticle($id)
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (Naming Rules)

(ShortVariable)